lark/parsers/earley.py

```diff
@@ -302,7 +302,7 @@ def parse(self, lexer, start):
         # this column. Find the item for the start_symbol, which is the root of the SPPF tree.
         solutions = [n.node for n in columns[-1] if n.is_complete and n.node is not None and n.s == start_symbol and n.start == 0]
         if not solutions:
-            expected_terminals = [t.expect for t in to_scan]
+            expected_terminals = [t.expect.name for t in to_scan]
```

Comment:
Why did you add this? I think you're breaking `match_examples()` with this.

Reply:
This is the same behaviour as displayed in line 247 for `UnexpectedToken`. inside of `UnexpectedEOF.__init__` we are just using `.name` anyway.

Reply:
Ah, okay. I confused it with a different parameter.